pkgs/development/interpreters/python/default.nix

```diff
@@ -71,9 +70,7 @@
             overrides
           ]);
           aliases = self: super: lib.optionalAttrs config.allowAliases (import ../../../top-level/python-aliases.nix lib self super);
-        in lib.makeScopeWithSplicing
-          splicePackages
-          newScope
+        in makeScopeWithSplicing
           otherSplices
```

Comment:
I think `topProjectedSpliced` can be used here as well.

Reply:
it's fine the way it is with `pythonOnXForX)